spec/requests/objects_controller_primary_moab_spec.rb

```diff
@@ -1,32 +1,48 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+
 RSpec.describe ObjectsController, type: :request do
+  let(:pres_obj) { primary_moab.preserved_object }
   let(:primary_moab) do
-    create(:complete_moab).tap { |cm| PreservedObjectsPrimaryMoab.create(complete_moab: cm, preserved_object: cm.preserved_object) }
+    create(:complete_moab) do |cm|
```

Comment:
`#tap` not needed here since FactoryBot's `create` method takes an optional block.